### ISSUE TITLE
Fix admin boundaries gaps

### ISSUE THREAD

**Issue by vgeorge** [earth-genome/foodtwin-global-app#17]
The admin boundaries on the staging website have small gaps.

The file `admin_polygon.gpkg` in the `v2` folder is a simplified version of the [GADM world database](https://gadm.org/download_world.html). The simplification process hasn't maintained topology, which is required to keep the borders in place.

We need to check if GADM provides simplified versions for subnational boundaries or if we can simplify it ourselves while maintaining topology.

For now, I suggest we use the original non-scaled version in the seed process until an enhanced solution is found.

cc @oliverroick @wrynearson

**Comment by wrynearson** [comment]
@vgeorge I think this has been fixed?

**Comment by oliverroick** [comment]
I don't think this has been actually fixed, but we changed the colours of the admin boundaries and added some transparency to make it less jarring.
